### PR TITLE
Small fixes to spanish translation

### PR DESCRIPTION
**Pull request by lozalojo**

### CODE REVIEW

File: lang/es.json [hunk 1]
```diff
@@ -13,17 +13,17 @@
     "WITCHER.Weapon.Reliability": "Fiab.",
     "WITCHER.Weapon.MaxReliability": "Max. Fiab.",
     "WITCHER.Weapon.Hands": "Manos",
-    "WITCHER.Weapon.Concealement": "CD.",
+    "WITCHER.Weapon.0": "CD.",
```
Comment (AnthonyMonette) [inline lang/es.json:16]: This must stays Concealement

**Review by AnthonyMonette — CHANGES_REQUESTED** [review]
The Localization has been completed for the system. But there is a lot of localization missing for Spanish.  It would be nice if you could complete it ! thanks

**Comment by lozalojo** [comment]
Sorry, I am not the one translating, just helping a friend which does not know git, just validated the es.json file and sent the PR, I will ask him to complete the translation, check it and send the PR again, but looks like there are a lot of keys missing, it will take some time.

**Comment by lozalojo** [comment]
@AnthonyMonette , included all keys in the translation file es.json, its ready now

**Pull request merged**